spesifikasjoner/afpant/afpant-kjoepekontrakt/README.md

```diff
@@ -1,9 +1,12 @@
 Kjøpekontrakt
 =============
 
-# Dokumentasjon
+Kjøpekontrakt eller informasjon fra kjøpekontrakten skal kunne oversendes fra megler til bank.
 
-* [Kjøpekontrakt løsningsbeskrivelse](kjoepekontrakt-loesningsbeskrivelse.md) - funksjonell løsningsbeskrivelse   
-  * Endringer i den funksjonelle løsningsbeskrivelsen gjøres her. Se [opprinnelig funksjonell løsningsbeskrivelse](Digital&#32;eiendomshandel&#32;-&#32;Løsningsbeskrivelse&#32;Kjøpekontrakt&#32;v1.0&#32;-&#32;2020-11-25.pdf)
-* [Kjøpekontrakt teknisk-beskrivelse](./kjoepekontrakt-teknisk-beskrivelse.md) inneholder tekniske informasjon om hvordan realisere meldingstype vedr. kjøpekontrakt
+Når megler har oversendt kjøpekontrakt, og det gjøres endringer i kjøpekontrakten eller legges til informasjon i meglersystemet, skal megler sende melding til banken med  oppdatert informasjon. 
```

Comment:
skal vi evt justere til "legges til eller endres informasjon knyttet til kjøpekontrakt i meglersystemet" etc? kan være at kun "legges til" er greit nok forøvig.